sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceivedMessage.java

```diff
@@ -238,25 +266,30 @@ public OffsetDateTime getExpiresAt() {
     /**
      * Gets the label for the message.
      *
+     * <p>
+     * This property enables the application to indicate the purpose of the message to the receiver in a standardized
+     * fashion, similar to an email subject line. The mapped AMQP property is "subject".
+     * </p>
+     *
      * @return The label for the message.
+     * @see ServiceBusMessage#getSubject()
      */
     public String getLabel() {
```

Comment:
This is also, I think we missed it,
This should get `getSubgect`. 
Just to verify this is .Net API View https://apiview.dev/Assemblies/Review/b5d7e3304000449c859c2016f718db52#Azure.Messaging.ServiceBus.ServiceBusReceivedMessage